dist/systemd/system/zincati.service

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=Zincati update agent
```

Comment:
This is minor, though I think the convention leans a bit more on the description being Title Case?